integration/log_rotation_test.go

```diff
@@ -111,6 +120,13 @@ func (s *LogRotationSuite) TestTraefikLogRotation(c *check.C) {
 	c.Assert(lineCount, checker.GreaterOrEqualThan, 7)
 }
 
+func logAccessLogFile(c *check.C, fileName string) {
+	output, err := ioutil.ReadFile(fileName)
+	c.Assert(err, checker.IsNil, check.Commentf("error opening access log file %s: %s", fileName, err))
```

Comment:
AFAIR the error from `ioutil.ReadFile` will already contain the file name and detailed error message.

Reply:
Good point. I removed the `Commentf`.